horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m

```diff
@@ -174,79 +174,11 @@
             % 8: 3xNpix array
```

Comment:
Shouldn't be included in this PR

Reply:
That's the conflict resolution (and subtle bug in init)

Reply:
It still shouldn't be included in this PR

Reply:
there are no other way of resolving conflicts

Reply:
```
git checkout master horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m
git commit --amend
git push -f
```

Reply:
So, I would be pushing to master invalid (outdated) code? I do not want this -- I will certainly forget to change it to recent version

Reply:
Where are you pushing outdated code to master? What are you talking about? This pulls the original file from master and puts it in this branch. If you need to make the change of `sqw_fopen` to the file, then you can do so before you commit. If you want to keep the other changes then you can check this filee out to another branch and open that as a PR.

Reply:
I have no idea what the problem is, but I've seen the conflict in PixelDataFileBacked and resolved it with the latest changes to code.

Reply:
Because it's a significant change not relevant to this PR

Reply:
If I see the issue, I am fixing it whatever possible and convenient. The relevance is very relative thing.